pratus.go

```diff
@@ -15,8 +15,13 @@ import (
 func getPRState(owner string, repo string, number int) (state string, statuses []github.RepoStatus, err error) {
 
 	ctx := context.Background()
+	token := os.Getenv("GITHUB_TOKEN")
```

Comment:
It would be nicer to do this in the main function so that this doesn't need to continuously look it up from the environment and so that it fails fast. 

```
Checking status of pull request 239 in elastic/helm-charts every 60 seconds
GITHUB_TOKEN must be set, aborting.
```